### PR TITLE
Update integration.sh to use standardized logic

### PR DESCRIPTION
**Pull request by aaron-lane**
This branch updates integration.sh to use the standardized logic from other modules.

### CODE REVIEW

File: test/integration.sh [hunk 1]
```diff
@@ -14,34 +14,49 @@
...
-	exit 1
-fi
+  # gcloud variables
+  export CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE="${tmpfile}"
+  # Application default credentials (Terraform google provider and inspec-gcp)
+  export GOOGLE_APPLICATION_CREDENTIALS="${tmpfile}"
+
+  # Terraform variables
+  export TF_VAR_project_id="${PROJECT_ID}"
+	export TF_VAR_region="${REGION:-us-east1}"
```
Comment (adrienthebo) [inline test/integration.sh]:
```suggestion
  export TF_VAR_region="${REGION:-us-east1}"
```

File: test/integration.sh [hunk 2]
```diff
@@ -14,34 +14,49 @@
...
+	export TF_VAR_zone="${ZONE:-us-east1-b}"
```
Comment (adrienthebo) [inline test/integration.sh]:
```suggestion
  export TF_VAR_zone="${ZONE:-us-east1-b}"
```

File: test/integration.sh [hunk 3]
```diff
@@ -14,34 +14,49 @@
...
 
-DELETE_AT_EXIT="$(mktemp -d)"
-finish() {
-	[[ -d "${DELETE_AT_EXIT}" ]] && rm -rf "${DELETE_AT_EXIT}"
+  # Setup environment variables
+  setup_environment
+  set -x
+
+  # Execute the test lifecycle
+  kitchen verify
```
Comment (adrienthebo) [inline test/integration.sh]: Should this be preceded with a `kitchen create` and `kitchen converge`?

Comment (aaron-lane) [reply test/integration.sh]: `kitchen verify` will invoke all preceding commands in the workflow if they have not been executed.

**Pull request merged**